Documentation/DocuBlocks/Rest/Replication/get_api_replication_revisions_tree.md

```diff
@@ -0,0 +1,66 @@
+
+@startDocuBlock get_api_replication_revisions_tree
+@brief retrieves the Merkle tree associated with the collection
+
+@RESTHEADER{GET /_api/replication/revisions/tree, Return Merkle tree for a collection,handleCommandRevisionTree}
+
+@RESTQUERYPARAMETERS
+
+@RESTQUERYPARAM{collection,string,required}
+The name or id of the collection to query.
+
+@RESTQUERYPARAM{batchId,number,required}
+The id of the snapshot to use
+
+@RESTDESCRIPTION
+Returns the Merkle tree from the collection.
+
+The result will be JSON/VelocyPack in the following format:
+```
+{
+  version: <Number>
+  maxDepth: <Number>,
+  rangeMin: <String, revision>,
+  rangeMax: <String, revision>,
+  nodes: [
+    { count: <Number>, hash: <String, revision> },
+    { count: <Number>, hash: <String, revision> },
+    ...
+    { count: <Number>, hash: <String, revision> }
+  ]
+}
+```
+
+At the moment, there is only one version, 1, so this can safely be ignored for
+now.
+
+Each `<String, revision>` value type is a 64-bit value encoded as a string of
+11 characters, using the same encoding as our document `_rev` values. The
+reason for this is that 64-bit values cannot necessarily be represented in full
+in JavaScript, as it handles all numbers as floating point, and can only
+represent up to `2^53-1` faithfully.
+
+The node count should correspond to a full tree with the given `maxDepth`.
```

Comment:
We could add a paragraph which describes how the tree is folded into a single array of "nodes". Any user of this must know, so we might as well define it in the documentation. With the `version` field we are safe for future changes. My preference would be to add such a description.